jetcd-core/src/main/java/io/etcd/jetcd/support/ProtoRequestMapper.java

```diff
@@ -0,0 +1,73 @@
+package io.etcd.jetcd.support;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+
+import io.etcd.jetcd.ByteSequence;
+import io.etcd.jetcd.Util;
+import io.etcd.jetcd.api.DeleteRangeRequest;
+import io.etcd.jetcd.api.PutRequest;
+import io.etcd.jetcd.api.RangeRequest;
+import io.etcd.jetcd.options.DeleteOption;
+import io.etcd.jetcd.options.GetOption;
+import io.etcd.jetcd.options.OptionsUtil;
+import io.etcd.jetcd.options.PutOption;
+
+import com.google.protobuf.ByteString;
+
+import static io.etcd.jetcd.options.OptionsUtil.toRangeRequestSortOrder;
+import static io.etcd.jetcd.options.OptionsUtil.toRangeRequestSortTarget;
+
+public class ProtoRequestMapper {
```

Comment:
this can be made final

Reply:
Also, it can be moved to the same package as the `Util` class

Reply:
the main one - io.etcd.jetcd? Ok, it'll help with additional boxing and  ByteSequence.toByteString (that is package private)

Reply:
yes, we may think to have better api vs impl separation in the future like when we can use sealed classes/interfaces